src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -466,7 +466,6 @@ private function addField(\DOMNode $node)
             $this->set(new Field\ChoiceFormField($node));
         } elseif ('input' == $nodeName && 'radio' == strtolower($node->getAttribute('type'))) {
             // there may be other fields with the same name that are no choice
-            // fields already registered (see https://github.com/symfony/symfony/issues/11689)
```

Comment:
Removing half of the comment is wrong (both lines are part of the same message between, making it broken)